userspace/libsinsp/container.cpp

```diff
@@ -83,6 +83,58 @@ const sinsp_container_info::container_mount_info *sinsp_container_info::mount_by
 	return NULL;
 }
 
+std::string sinsp_container_info::normalize_healthcheck_arg(const std::string &arg)
+{
+	std::string ret = arg;
+
+	if(ret.empty())
+	{
+		return ret;
+	}
+
+	// Remove pairs of leading/trailing " or ' chars, if present
+	while(ret.front() == '"' || ret.front() == '\'')
+	{
+		if(ret.back() == ret.front())
+		{
+			ret.pop_back();
+			ret.erase(0, 1);
+		}
+	}
+
+	return ret;
+}
+
+void sinsp_container_info::parse_healthcheck(const Json::Value &healthcheck_obj)
+{
+	if(!healthcheck_obj.isNull())
```

Comment:
what if it's null? does it make sense to log it?

Reply:
Nope, it just means that the container doesn't have any health check configured.

Reply:
oh gotcha!